# Import Libraries

In [1]:
import pandas as pd
import numpy as np
import os

## Viewing the Dataset

In [2]:
pwd = os.getcwd() 
filepath = pwd + "/android-games.csv"
df = pd.read_csv(filepath)

lets read and check our dataset

In [6]:
df.shape

(1730, 15)

* so we have 1730 rows and 15 columns working with
* lets take a look at the columns and summary info.

In [10]:
df.columns

Index(['rank', 'title', 'total ratings', 'installs', 'average rating',
       'growth (30 days)', 'growth (60 days)', 'price', 'category',
       '5 star ratings', '4 star ratings', '3 star ratings', '2 star ratings',
       '1 star ratings', 'paid'],
      dtype='object')

ok maybe we should sample our dataset to explore what it contains.
lets sample the first 5 rows.

In [8]:
df.sample(n=5)

rank                                             title  total ratings  \
1164    39                   MARVEL Strike Force - Squad RPG         553833   
1207    82  MARVEL Puzzle Quest: Join the Super Hero Battle!         351850   
83      84                                       Injustice 2         809654   
1163    38                                    Dragon Village         556175   
311     12                                         Ludo STAR         769196   

     installs  average rating  growth (30 days)  growth (60 days)  price  \
1164   10.0 M               3               1.6               3.3    0.0   
1207   10.0 M               4               0.3               0.6    0.0   
83     10.0 M               4               0.5               1.3    0.0   
1163    5.0 M               4               0.1               0.2    0.0   
311    10.0 M               4               2.1               4.5    0.0   

               category  5 star ratings  4 star ratings  3 star ratings  \
1164  GAME ROLE PLAYING          281401           76022           42302   
1207  GAME ROLE PLAYING          221235           57641           24241   
83          GAME ACTION          537380          104057           50801   
1163  GAME ROLE PLAYING          407087           51168           33739   
311          GAME BOARD          597616           56899           32850   

      2 star ratings  1 star ratings   paid  
1164           49578          104528  False  
1207           12944           35787  False  
83             25962           91451  False  
1163           13251           50928  False  
311            14912           66917  False

In [4]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1730 entries, 0 to 1729
Data columns (total 15 columns):
 #   Column            Non-Null Count  Dtype  
---  ------            --------------  -----  
 0   rank              1730 non-null   int64  
 1   title             1730 non-null   object 
 2   total ratings     1730 non-null   int64  
 3   installs          1730 non-null   object 
 4   average rating    1730 non-null   int64  
 5   growth (30 days)  1730 non-null   float64
 6   growth (60 days)  1730 non-null   float64
 7   price             1730 non-null   float64
 8   category          1730 non-null   object 
 9   5 star ratings    1730 non-null   int64  
 10  4 star ratings    1730 non-null   int64  
 11  3 star ratings    1730 non-null   int64  
 12  2 star ratings    1730 non-null   int64  
 13  1 star ratings    1730 non-null   int64  
 14  paid              1730 non-null   bool   
dtypes: bool(1), float64(3), int64(8), object(3)
memory usage: 191.0+ KB
